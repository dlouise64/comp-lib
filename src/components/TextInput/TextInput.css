.textInput {
  border: 1px solid #ccc;
  border-radius: 4px;
  padding: 10px;
  font-size: 16px;
}
