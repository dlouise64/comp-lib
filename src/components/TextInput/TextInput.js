import React from 'react';
// import PropTypes from 'prop-types';

import './TextInput.css';

class TextInput extends React.Component {
  render() {
    return (
      <div className="textInput">TextInput</div>
    );
  }
}

TextInput.propTypes = {
};

TextInput.defaultProps = {
}

export default TextInput;
