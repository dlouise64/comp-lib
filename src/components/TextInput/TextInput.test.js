import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from './TextInput';

test('<TextInput /> renders correctly', () => {
  const tree = renderer.create(<TextInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
