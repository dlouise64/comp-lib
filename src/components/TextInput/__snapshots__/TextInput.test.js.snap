// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextInput /> renders correctly 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
      "marginBottom": 16,
      "position": "relative",
    }
  }
>
  <label
    className="label"
    htmlFor="my_id"
    style={
      Object {
        "display": "inline-block",
        "marginRight": "10px",
      }
    }
  >
    the label
     
  </label>
  <input
    className="textInput"
    id="my_id"
    name="my_name"
    onChange={[Function]}
    type="text"
  />
</div>
`;
